### ISSUE TITLE
When convert a ml using backend of pytorch, how can I save the converted model into pytorch format?

### ISSUE THREAD

**Issue by GoHard2023** [microsoft/hummingbird#641]
As titled. Is there a way I can save a converted model as pytorch format?

**Comment by interesaaat** [comment]
You can find the tests [here](https://github.com/microsoft/hummingbird/blob/9e59b1e25947ac43a531115852c6b399c0ed5433/tests/test_backends.py#L100) showing how to do it with the different backends

**Comment by GoHard2023** [comment]
> You can find the tests [here](https://github.com/microsoft/hummingbird/blob/9e59b1e25947ac43a531115852c6b399c0ed5433/tests/test_backends.py#L100) showing how to do it with the different backends

Thank you for replying! I walked over the test cases. I did not find the test code to save converted model as pytorch format. Maybe I was not clear enough. I want to the saved model can be loaded by some pytorch code.

Thanks.

**Comment by interesaaat** [comment]
When you save a model we create a zip file containing a bunch of metadata and the model in the pytorch format (if you chose that as backend). 

If you want just to save and load the pytorch model you can just do `hb_model.model.save(path)` and `torch.load(path)`. (the `model` attribute is a pytorch model). Doing this way however you will lose the Hummingbird metadata so you will have to convert the input data yourself, for example.

**Comment by GoHard2023** [comment]
> When you save a model we create a zip file containing a bunch of metadata and the model in the pytorch format (if you chose that as backend).
> 
> If you want just to save and load the pytorch model you can just do `hb_model.model.save(path)` and `torch.load(path)`. (the `model` attribute is a pytorch model). Doing this way however you will lose the Hummingbird metadata so you will have to convert the input data yourself, for example.

Thanks for the tip! `hb_model.model.save(path)` does not work out for me. I used `torch.save(hb_model.model.state_dict()) `to save the model as pytorch match I guess,

**Comment by GoHard2023** [comment]
By the way, is there a way that I can select only probability when converting random forest to pytorch in hummingbird?

**Comment by interesaaat** [comment]
No. You can run ‘predict_prob’ to get the probabilities, but conversion is
the same.
